src/panels/config/areas/dialog-area-registry-detail.ts

```diff
@@ -3,19 +3,30 @@ import "@polymer/paper-input/paper-input";
 import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
 import { property, state } from "lit/decorators";
 import { fireEvent } from "../../../common/dom/fire_event";
-import { navigate } from "../../../common/navigate";
+import "../../../components/ha-alert";
 import "../../../components/ha-dialog";
+import "../../../components/ha-picture-upload";
+import type { HaPictureUpload } from "../../../components/ha-picture-upload";
 import { AreaRegistryEntryMutableParams } from "../../../data/area_registry";
+import { CropOptions } from "../../../dialogs/image-cropper-dialog/show-image-cropper-dialog";
 import { PolymerChangedEvent } from "../../../polymer-types";
 import { haStyleDialog } from "../../../resources/styles";
 import { HomeAssistant } from "../../../types";
 import { AreaRegistryDetailDialogParams } from "./show-dialog-area-registry-detail";
 
+const cropOptions: CropOptions = {
```

Comment:
Can we force `aspectRatio` of 16:9? or maybe even 16:5 ?

Reply:
I didn't do that as we are not sure how we are going to use it yet, and it would give us the possibility to crop it when used, but that does mean the user can't pick what part is shown...

Reply:
I think that we should force an aspect ratio from the start, it needs to be a landscape shape. Let's pick one :)